ext/CMakeLists.txt

```diff
@@ -111,7 +111,7 @@ add_subdirectory(drmgr)
 add_subdirectory(drx)
```

Comment:
(FYI, it is possible to view the .svg by clicking on the Rich Diff button in the upper right, which I had never realized before.)